gordo_components/model/model.py

```diff
@@ -1,18 +0,0 @@
-# -*- coding: utf-8 -*-
-
```

Comment:
Did you delete this file?

Reply:
No, thanks for the reminder, I will remove it.